integration/test_collection_config.py

```diff
@@ -400,6 +409,7 @@ def test_collection_config_update(collection_factory: CollectionFactory) -> None
     assert config.inverted_index_config.stopwords.removals == ["the"]
 
     # assert config.replication_config.factor == 2
+    # assert config.replication_config.async_enabled is True
```

Comment:
why commented out?

Reply:
This was commented out because replication couldn't be updated. I will experiment whether that feature is still missing now

Reply:
if it is not working let's also comment out the update function

Reply:
It works fine, so I've uncommented it